OpenRA.Mods.Common/Effects/RepairIndicator.cs

```diff
@@ -19,18 +19,18 @@ namespace OpenRA.Mods.Common.Effects
 	class RepairIndicator : IEffect
 	{
 		readonly Actor building;
-		readonly string palettePrefix;
 		readonly Animation anim;
 		readonly RepairableBuilding rb;
+
 		int shownPlayer = 0;
+		PaletteReference palette;
```

Comment:
This is only ever used as a local variable, so it shouldn't be stored as a field.